src/iocore/cache/RamCacheLRU.cc

```diff
@@ -54,7 +54,7 @@ struct RamCacheLRU : public RamCache {
   void init(int64_t max_bytes, StripeSM *stripe) override;
 
   // private
-  std::vector<bool> *seen = nullptr;
+  std::vector<bool> seen{};
```

Comment:
The `{}` is actually redundant here, but doesn't hurt.

Reply:
I removed `{}` at [0b4de59](https://github.com/apache/trafficserver/pull/12229/commits/0b4de5905caefe754f433476fdb3c39159057eb1).